signer/src/stacks/api.rs

```diff
@@ -1574,7 +1504,84 @@ fn extract_signatures_required(value: Value) -> Result<Option<u16>, Error> {
     }
 }
 
+#[derive(Debug, Deserialize)]
+struct GetTenureHeadersLightApiResponse {
+    #[serde(rename = "consensus_hash")]
+    pub _consensus_hash: String,
+    pub burn_block_height: u64,
+    pub burn_block_hash: String,
```

Comment:
We should be able to have these types deserialize to `BitcoinBlockHeight` and `BitcoinBlockHash`